newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/Utils.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+import { localStorageLocalStatsPrefix } from '../../../../Utils/Analytics/LocalStats';
+
+export const getGetStartedSectionViewCount = (): number => {
```

Comment:
wondering if those functions should be in the `LocalStats` file to avoid having multiple places where we manipulate the localstorage